codalab/apps/web/static/js/search/ws_actions.js

```diff
@@ -98,6 +98,15 @@ var WorksheetActions =  function() {
                                 if (data.data.stdout){
                                     console.log(data.data.stdout);
                                     term.echo(data.data.stdout);
+                                    var references = data.data.structured_result['refs'];
+                                    Object.keys(references).forEach(function(k) {
+                                        $(".terminal-output div div:contains("+ k + ")").html(
+                                            function(idx, html) {
+                                                link = '/' + references[k]['type'] + 's/' + references[k]['uuid'];
```

Comment:
Use `var link` (to avoid setting `link` as a global variable on the `window` object).